packages/core/components/StatusMessage/StatusMessage.module.css

```diff
@@ -4,6 +4,7 @@
     width: 40vw; 
     height: 100%;
     left: 30px;
+    /* FluentUI modal z-index is set to 1000000 */
```

Comment:
Is this comment right seems like 1000 not 1000000?

Reply:
Yes, this is for the status message component. The modal z-index is set by fluent, not us, so this comment is intended to note that the status message will be covered by the modal overlay. I can clarify or remove this comment